doc/source/quickstart.rst

```diff
@@ -70,25 +40,49 @@ scripts and XML files to configure a case (see below) for the requested
 resolution, component set, and machine. **create_newcase** has several
 required arguments (invoke **create_newcase --help** for help).
 
-If running on a supported machine, (``$MACHINE``), that machine will
+The ``$PROJECT`` variable must be specified in either your 
+shell environment or as the ``--project`` argument to **create_newcase**.
+
+If running on a supported machine, that machine will
 normally be recognized automatically and therefore it is not required
 to specify the ``--machine`` argument to **create_newcase**, then
 invoke **create_newcase** as follows:
 
 .. code-block:: console
 
-    create_newcase --case $CASEROOT --compset $COMPSET --res $GRID 
+    ./create_newcase --case $CASEROOT --compset $COMPSET --res $GRID
+
+An example on NCAR machine cheyenne:
+
+.. code-block:: console
+
+    ./create_newcase --case /glade/scratch/[username]/cases/b.e20.B1850.f19_g17.test --compset B1850 --res f19_g17 
+
+where:
+
+-  ``$COMPSET`` refers to the `component set <http://www.cesm.ucar.edu/models/cesm2.0/config/compsets.html>`_.
+
+-  ``$GRID`` refers to the model `resolution <http://www.cesm.ucar.edu/models/cesm2.0/config/grids.html>`_.
+
+-  ``$CASEROOT`` refers to the full pathname of the root directory where the
+   case (``$CASE``) will be created. In the example above, the ``$CASEROOT = /glade/scratch/[username]/cases/A.f19_g17.test``
```

Comment:
When describing what ``$CASEROOT`` is, either drop the "the", or spell out, "the ``$CASEROOT`` is ..."

Also, this path is wrong for the example you gave. Also, as above, change [username] to ``$USER``?

Reply:
fixed.